types/params.go

```diff
@@ -83,6 +88,23 @@ func (a ABCIParams) VoteExtensionsEnabled(h int64) bool {
 	return a.VoteExtensionsEnableHeight <= h
 }
 
+// FeatureParams configure parameters of different features of CometBFT.
+type FeatureParams struct {
+	VoteExtensionsEnableHeight *int64 `json:"vote_extensions_enable_height"`
+	PbtsEnableHeight           *int64 `json:"pbts_enable_height"`
+}
+
+// PbtsEnabled returns true if PBTS are enabled at height h and false otherwise.
+func (p FeatureParams) PbtsEnabled(h int64) bool {
+	if h < 1 {
```

Comment:
Can this happen? The validation shouldn't allow for this.

Reply:
It should never happen.